servers/src/test/java/tachyon/master/file/meta/InodeTreeTest.java

```diff
@@ -173,11 +174,12 @@ public void createPathTest() throws Exception {
     CommonUtils.sleepMs(10);
 
     // creating the directory path again results in no new inodes.
-    createResult = mTree.createPath(NESTED_URI, sNestedDirectoryOptions);
-    modified = createResult.getModified();
-    created = createResult.getCreated();
-    Assert.assertEquals(0, modified.size());
-    Assert.assertEquals(0, created.size());
+    try {
+      createResult = mTree.createPath(NESTED_URI, sNestedDirectoryOptions);
+    } catch (FileAlreadyExistsException faee) {
+      Assert.assertEquals(faee.getMessage(),
```

Comment:
We cannot use `ExpectedException` here because it will skip the rest of the test. I added a Assert to make sure the exception is thrown though.